server/src/service.rs

```diff
@@ -134,35 +139,52 @@ impl Service {
             manager: Default::default(),
             connection: connection.clone(),
         };
-        let collections = service.fetch_collections().await;
 
+        // Load login/default collection
+        let mut login: Option<Collection> = None;
+        if secret.is_some() {
+            login = Some(service.fetch_login(secret).await?);
+        }
+        // Load session collection
+        let session = service.fetch_session().await?;
+
+        // Dispatch Service
         object_server
             .at(oo7::dbus::api::Service::PATH.as_deref().unwrap(), service)
             .await?;
-
-        for collection in collections {
-            object_server
-                .at(collection.path().clone(), collection)
-                .await?;
+        // Dispatch login collection
```

Comment:
Useless comment, same with the one below

Reply:
Yup, removed.